bin/compile

```diff
@@ -65,7 +65,14 @@ fi
 
 set-env PATH '/app/.indyno/vendor/redis/bin:$PATH'
 PASSWORD=`openssl rand -hex 16`
-export REDIS_URL="redis://h:$PASSWORD@localhost:6379/"
+
+# placeholder username h was added for compatibility with old Redis clients (https://devcenter.heroku.com/changelog-items/1932) and is incompatible with Redis 6 AUTH
+if [[ "${VERSION}" == 6* ]]; then
```

Comment:
```suggestion
if [[ "${VERSION}" == 6.* ]]; then
```

Reply:
Haha, thank you - that's completely true. I was thinking about that use case explicitly and as you say we'd expand it already.